constants/Colors.ts

```diff
@@ -13,10 +13,15 @@ export default {
     tabIconSelected: tintColorLight,
   },
   dark: {
-    text: '#fff',
-    background: '#000',
-    tint: tintColorDark,
+    // text: '#fff',
+    // background: '#000',
+    // tint: tintColorDark,
+    // tabIconDefault: '#ccc',
+    // tabIconSelected: tintColorDark,
+    text: '#000',
```

Comment:
delete the comments?

Reply:
I kept this in case we need dark mode in the future lol